k2/python/csrc/torch/doc/any.h

```diff
@@ -0,0 +1,199 @@
+
+
+#ifndef K2_PYTHON_CSRC_TORCH_DOC_ANY_H_
+#define K2_PYTHON_CSRC_TORCH_DOC_ANY_H_
+
+namespace k2 {
+
+static constexpr const char *kRaggedAnyInitDataDoc = R"doc(
+Create a ragged tensor with two axes.
```

Comment:
I am trying to put the documentation in C++ headers.
Users can view the help information in a usual way, i.e.,
```python
>>> import k2.ragged
>>> help(k2.ragged.Tensor.__init__)
```

The output is given below:
![Screen Shot 2021-08-28 at 18 14 17](https://user-images.githubusercontent.com/5284924/131214600-b4d13149-f4d7-499f-88b4-383d54d4f515.png)

---

Also, the doc is going to be hosted at
https://k2-fsa.github.io/k2/index.html

The reason is that now the doc depends on the C++ source code and we have to compile
 `_k2` to generate the doc.

---
A preview is available at 
https://csukuangfj.github.io/k2/python_api/tensor.html#k2.ragged.Tensor

Reply:
The doc is easier to discover as it is bound to the actual class. There is no need to create a fake class just for documentation purposes.